benchmark/blankenbach/Makefile

```diff
@@ -0,0 +1,38 @@
+.PHONY: all
```

Comment:
This whole Makefile is a bit obscure. A couple of lines at the top providing an overview of what it does would be helpful.